Platform support for Chrome hardware
Say Y here to get to see options for platform support for
various Chromebooks and Chromeboxes. This option alone does
not add any kernel code.

If you say N, all options in this submenu will be skipped and disabled.
Depends: X86 || ARM || ARM64 || COMPILE_TEST
---
--- /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config.orig	2022-10-31 13:57:59.705695527 -0400
+++ /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config	2022-11-01 16:20:36.975610301 -0400
@@ -3926,7 +3926,6 @@
 # CONFIG_KEYBOARD_SUNKBD is not set
 CONFIG_KEYBOARD_TM2_TOUCHKEY=m
 # CONFIG_KEYBOARD_XTKBD is not set
-CONFIG_KEYBOARD_CROS_EC=m
 CONFIG_KEYBOARD_CYPRESS_SF=m
 CONFIG_INPUT_MOUSE=y
 CONFIG_MOUSE_PS2=y
@@ -4413,7 +4412,6 @@
 # Other I2C/SMBus bus drivers
 #
 CONFIG_I2C_MLXCPLD=m
-CONFIG_I2C_CROS_EC_TUNNEL=m
 CONFIG_I2C_VIRTIO=m
 # end of I2C Hardware Bus support
 
@@ -4717,8 +4715,6 @@
 # CONFIG_BATTERY_GOLDFISH is not set
 CONFIG_BATTERY_RT5033=m
 # CONFIG_CHARGER_RT9455 is not set
-CONFIG_CHARGER_CROS_USBPD=m
-CONFIG_CHARGER_CROS_PCHG=m
 CONFIG_CHARGER_BD99954=m
 CONFIG_BATTERY_SURFACE=m
 CONFIG_CHARGER_SURFACE=m
@@ -5111,7 +5107,6 @@
 CONFIG_MFD_BD9571MWV=m
 CONFIG_MFD_AXP20X=y
 CONFIG_MFD_AXP20X_I2C=y
-CONFIG_MFD_CROS_EC_DEV=m
 # CONFIG_MFD_MADERA is not set
 # CONFIG_PMIC_DA903X is not set
 # CONFIG_MFD_DA9052_SPI is not set
@@ -5311,7 +5306,6 @@
 # CONFIG_CEC_PIN_ERROR_INJ is not set
 CONFIG_MEDIA_CEC_SUPPORT=y
 CONFIG_CEC_CH7322=m
-# CONFIG_CEC_CROS_EC is not set
 CONFIG_CEC_GPIO=m
 CONFIG_CEC_SECO=m
 CONFIG_CEC_SECO_RC=y
@@ -6596,7 +6590,6 @@
 CONFIG_SND_SOC_AMD_CZ_DA7219MX98357_MACH=m
 CONFIG_SND_SOC_AMD_CZ_RT5645_MACH=m
 CONFIG_SND_SOC_AMD_ACP3x=m
-CONFIG_SND_SOC_AMD_RV_RT5682_MACH=m
 CONFIG_SND_SOC_AMD_RENOIR=m
 CONFIG_SND_SOC_AMD_RENOIR_MACH=m
 CONFIG_SND_SOC_AMD_ACP5x=m
@@ -6784,7 +6777,6 @@
 CONFIG_SND_SOC_AW8738=m
 CONFIG_SND_SOC_BD28623=m
 CONFIG_SND_SOC_BT_SCO=m
-CONFIG_SND_SOC_CROS_EC_CODEC=m
 # CONFIG_SND_SOC_CS35L32 is not set
 # CONFIG_SND_SOC_CS35L33 is not set
 CONFIG_SND_SOC_CS35L34=m
@@ -7042,7 +7034,6 @@
 CONFIG_HID_HOLTEK=m
 CONFIG_HOLTEK_FF=y
 CONFIG_HID_VIVALDI_COMMON=m
-# CONFIG_HID_GOOGLE_HAMMER is not set
 CONFIG_HID_VIVALDI=m
 CONFIG_HID_GT683R=m
 CONFIG_HID_KEYTOUCH=m
@@ -7751,7 +7742,6 @@
 CONFIG_RTC_DRV_BQ4802=m
 CONFIG_RTC_DRV_RP5C01=m
 CONFIG_RTC_DRV_V3020=m
-CONFIG_RTC_DRV_CROS_EC=m
 
 #
 # on-CPU RTC drivers
@@ -8172,28 +8162,7 @@
 # CONFIG_SIEMENS_SIMATIC_IPC is not set
 CONFIG_WINMATE_FM07_KEYS=m
 CONFIG_PMC_ATOM=y
-CONFIG_CHROME_PLATFORMS=y
-CONFIG_CHROMEOS_ACPI=m
-CONFIG_CHROMEOS_LAPTOP=m
-CONFIG_CHROMEOS_PSTORE=m
-CONFIG_CHROMEOS_TBMC=y
-CONFIG_CROS_EC=m
-CONFIG_CROS_EC_I2C=m
-# CONFIG_CROS_EC_ISHTP is not set
-CONFIG_CROS_EC_SPI=m
-CONFIG_CROS_EC_LPC=m
-CONFIG_CROS_EC_PROTO=y
-CONFIG_CROS_KBD_LED_BACKLIGHT=m
-CONFIG_CROS_EC_CHARDEV=m
-CONFIG_CROS_EC_LIGHTBAR=m
-# CONFIG_CROS_EC_DEBUGFS is not set
-CONFIG_CROS_EC_SENSORHUB=m
-CONFIG_CROS_EC_SYSFS=m
-CONFIG_CROS_EC_TYPEC=m
-CONFIG_CROS_USBPD_LOGGER=m
-CONFIG_CROS_USBPD_NOTIFY=m
-CONFIG_CHROMEOS_PRIVACY_SCREEN=m
-# CONFIG_WILCO_EC is not set
+# CONFIG_CHROME_PLATFORMS is not set
 CONFIG_MELLANOX_PLATFORM=y
 CONFIG_MLXREG_HOTPLUG=m
 CONFIG_MLXREG_IO=m
@@ -8355,7 +8324,6 @@
 # CONFIG_EXTCON_RT8973A is not set
 # CONFIG_EXTCON_SM5502 is not set
 # CONFIG_EXTCON_USB_GPIO is not set
-CONFIG_EXTCON_USBC_CROS_EC=m
 CONFIG_EXTCON_USBC_TUSB320=m
 # CONFIG_MEMORY is not set
 CONFIG_IIO=m
@@ -8407,7 +8375,6 @@
 # CONFIG_FXLS8962AF_I2C is not set
 # CONFIG_FXLS8962AF_SPI is not set
 CONFIG_HID_SENSOR_ACCEL_3D=m
-CONFIG_IIO_CROS_EC_ACCEL_LEGACY=m
 CONFIG_IIO_ST_ACCEL_3AXIS=m
 CONFIG_IIO_ST_ACCEL_I2C_3AXIS=m
 CONFIG_IIO_ST_ACCEL_SPI_3AXIS=m
@@ -8541,10 +8508,6 @@
 # CONFIG_VZ89X is not set
 # end of Chemical Sensors
 
-CONFIG_IIO_CROS_EC_SENSORS_CORE=m
-CONFIG_IIO_CROS_EC_SENSORS=m
-CONFIG_IIO_CROS_EC_SENSORS_LID_ANGLE=m
-
 #
 # Hid Sensor IIO Common
 #
@@ -8744,7 +8707,6 @@
 # CONFIG_CM3323 is not set
 CONFIG_CM3605=m
 # CONFIG_CM36651 is not set
-CONFIG_IIO_CROS_EC_LIGHT_PROX=m
 CONFIG_GP2AP002=m
 # CONFIG_GP2AP020A00F is not set
 # CONFIG_SENSORS_ISL29018 is not set
@@ -8863,7 +8825,6 @@
 CONFIG_BMP280=m
 CONFIG_BMP280_I2C=m
 CONFIG_BMP280_SPI=m
-CONFIG_IIO_CROS_EC_BARO=m
 # CONFIG_DLHL60D is not set
 # CONFIG_DPS310 is not set
 # CONFIG_HID_SENSOR_PRESS is not set
@@ -8890,7 +8851,6 @@
 #
 # Proximity and distance sensors
 #
-CONFIG_CROS_EC_MKBP_PROXIMITY=m
 # CONFIG_ISL29501 is not set
 # CONFIG_LIDAR_LITE_V2 is not set
 CONFIG_MB1232=m
@@ -8946,7 +8906,6 @@
 CONFIG_PWM=y
 CONFIG_PWM_SYSFS=y
 CONFIG_PWM_CRC=y
-CONFIG_PWM_CROS_EC=m
 CONFIG_PWM_DWC=m
 CONFIG_PWM_LPSS=m
 CONFIG_PWM_LPSS_PCI=m
